cmake/package_helpers.cmake

```diff
@@ -29,43 +29,45 @@ endfunction()
 #   Create a new target and declare it as `IMPORTED` for libraries or `INTERFACE`
 #       for header only projects.
 #
-#   \param name             Name of external project
```

Comment:
Question: do we put a note somewhere that this macro can be simplified and we can use ALIAS whenever we upgrade to 3.11+? We could also just keep the current form I believe.

Reply:
I actually don't think it would be necessary because we don't need the additional call `add_library(new_target ALIAS project_name)` after calling `ginkgo_add_external_target` anymore.